src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentUtils.java

```diff
@@ -302,7 +351,32 @@ public static void writeACNVModeledSegmentFile(final File outFile,
                                 .append(String.format("%6.4f", segment.getMinorAlleleFractionPosteriorSummary().lower())
                                         .replaceAll("\\s+", ""))
                                 .append(String.format("%6.4f", segment.getMinorAlleleFractionPosteriorSummary().upper())
-                                        .replaceAll("\\s+","")));
+                                        .replaceAll("\\s+", "")));
+    }
+
+    /**
+     * Read ACNV file into ACNV modeled segments.
+     *
+     * @param acnvSegFile File in the ACNV format (*-final.seg).  Must be readable file and never {@code null}
+     * @return Never {@code null}.  Can be empty list if no entries exist in the input file.
+     */
+    public static List<ACNVModeledSegment> readACNVModeledSegmentFile(final File acnvSegFile) {
```

Comment:
Comment 1: Done.

Comment 2: I think the shuffling is only because I had to change a few private's to public, so I moved them appropriately.  Do you see counter-example?